### PR TITLE
Increase test coverage

### PR DESCRIPTION
**Pull request by jvdassen**
Hello everyone,

We have increase the scope of the unit testing in this project with focus on the `classify`, `math`, `process`, `simple` and `util` packages.
As you can see there were no modifications to the `src` of the project and we did not incorporate new dependencies as requested in [PR 1022](https://github.com/stanfordnlp/CoreNLP/issues/1022). 
All tests were written using the existing JUnit version and aligned to the formatting of the existing classes.

As requested in the [contribution guidelines](https://github.com/stanfordnlp/CoreNLP/blob/master/CONTRIBUTING.md) we are happy to declare this contribution under public domain.

### CODE REVIEW

**Comment by AngledLuffa** [comment]
I'm going to start going through this, but I'd appreciate it if you could squash some of the changes down and set your whitespace to use 2 spaces instead of tabs.

File: test/src/edu/stanford/nlp/simple/TokenTest.java [hunk 1]
```diff
@@ -79,13 +78,19 @@ public void testCharacterOffsetEnd() {
     }
 
     @Test
-    public void testWhiteSpace() {
+    public void testWhiteSpaceBefore() {
```
Comment (AngledLuffa) [inline test/src/edu/stanford/nlp/simple/TokenTest.java:81]: This really doesn't seem necessary

File: test/src/edu/stanford/nlp/dcoref/RulesTest.java [hunk 2]
```diff
@@ -20,12 +19,12 @@
   List<CoreLabel> MIBM = SentenceUtils.toCoreLabelList("MIBM");
 
   public void testIsAcronym() {
-    assertTrue(Rules.isAcronym(IBM, IBM2));
-    assertTrue(Rules.isAcronym(IBM2, IBM));
-    assertFalse(Rules.isAcronym(IBM, IBMM));
-    assertFalse(Rules.isAcronym(IBM2, IBMM));
-    assertFalse(Rules.isAcronym(IBM, MIBM));
-    assertFalse(Rules.isAcronym(IBM2, MIBM));
+    assertTrue("Acronym IMB -> International Business Machines",Rules.isAcronym(IBM, IBM2));
```
Comment (AngledLuffa) [inline test/src/edu/stanford/nlp/dcoref/RulesTest.java]: You get a line number for where the test fails so I don't really agree this is an mystery assertion.  If you want to gather up a bunch of these changes into a single change, though, I can merge it

**Comment by jvdassen** [comment]
Thanks for the feedback! Unfortunately well have to create a new PR to change the upstream branch to the one where we fixed and squashed our contributions!

**Pull request closed without merging**